ledlamp_bowtech.orogen

```diff
@@ -6,22 +6,19 @@ import_types_from "base"
 using_library "ledlamp_bowtech"
 using_library "iodrivers_base"
 
-using_task_library "iodrivers_base"  
+using_task_library "iodrivers_base"
 
 task_context "Task" do
     needs_configuration
+    subclasses "iodrivers_base::Task"
+
+    #List of all connected lamps
+    property("led_list", "std/vector<ledlamp_bowtech/LedLamp>").dynamic
```

Comment:
Does it really make any sense to have a dynamic property there ? The lamps won't appear and disappear magically.